homeassistant/components/camera/services.yaml

```diff
@@ -69,17 +69,33 @@ record:
       example: 4
 
 onvif_ptz:
```

Comment:
Please move this description to a new services.yaml file under the onvif integration package.

Reply:
it is what i had made in the first place but it had no effect, but maybe related to your previous comment about wrong domain usage

Reply:
Yes it was domain related, many thanks pointed it out to me !